compose/ui/ui/build.gradle

```diff
@@ -146,6 +146,7 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
         wasm()
 
         configureDarwinFlags()
+        configureTestsRunInIosSimulatorEnvironment()
```

Comment:
Why should it be explicit call for each module?

Reply:
It should be explicit call for the module that support tests with booted simulator. For now, only one module need this environment.